<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Desktop-22" failures="0" tests="3" name="Dashboard.Customer_Landing_Test_Cases.TC_07_TC_13_TC_15_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results" time="211.100" errors="1" timestamp="2025-07-30T18:48:43 IST" skipped="0">
  <testcase classname="Dashboard.Customer_Landing_Test_Cases.TC_07_TC_13_TC_15_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results" name="TC_07_Verify_the_timeline_for_SH_test_with_unconfirmed_results" time="6.837"/>
  <system-out/>
  <testcase classname="Dashboard.Customer_Landing_Test_Cases.TC_07_TC_13_TC_15_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results" name="TC_13_Verify_the_display_of_calendly_calendar_with_message_for_SH_tests" time="191.718"/>
  <system-out/>
  <testcase classname="Dashboard.Customer_Landing_Test_Cases.TC_07_TC_13_TC_15_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results" name="TC_15_Verify_the_dashboard_update_after_the_booked_appointment" time="12.545">
    <error message="Date &#039;August 1&#039; not found or not selectable in calendar." type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: Date 'August 1' not found or not selectable in calendar.
at Wrappers.CalendlyCalendar.selectDateOnly(CalendlyCalendar.java:61)
at Wrappers.CalendlyCalendar.selectDateAfterTwoDaysSkippingWeekends(CalendlyCalendar.java:41)
at Test.Dashboard.Actions_Class.Customer_Landing_Actions.verifySuccsessfulAppointmentBooking(Customer_Landing_Actions.java:809)
at Dashboard.Customer_Landing_Test_Cases.TC_07_TC_13_TC_15_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results.TC_15_Verify_the_dashboard_update_after_the_booked_appointment(TC_07_TC_13_TC_15_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results.java:29)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:136)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:658)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:219)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:923)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:192)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- TC_15_Verify_the_dashboard_update_after_the_booked_appointment -->
  <system-out/>
</testsuite> <!-- Dashboard.Customer_Landing_Test_Cases.TC_07_TC_13_TC_15_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results -->
